wrappers/modules/hm.nix

```diff
@@ -3,5 +3,13 @@ with lib; {
   options = {
     enable = mkEnableOption "nixvim";
     defaultEditor = mkEnableOption "nixvim as the default editor";
+
+    vimdiffAlias = mkOption {
```

Comment:
This could use `mkEnableOption`

Reply:
Yes we could, but it would force the description to start with "Whether to enable" which is not ideal.

Reply:
> which is not ideal

Not sure I agree "Whether to enable `vimdiff` alias" is not ideal, but I'll agree to disagree :grinning: